### DISCUSSION TITLE
Can I add few extensions to astroidframe.work

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#112] [General]
I have few extensions that I like to add to astroidframe.work
I volunteer to take care of creating backups of the website (akeeba Backup) and even maintaining a local copy (on my computer) as failsafe and to test Joomla updates or component additions before updating the live version (The professional way to do it)

**For now, I like to ADD**
- Akeeba backup core
-  JCE Pro version

If needed, I can also install Akeeba Admin Pro for security and optimization.

Will need full access.

———————————————————————————
**Also I get this error in the editor ?**
Tested in Chrome/Safari

![Screen Shot 2022-03-07 at 10 25 14 AM](https://user-images.githubusercontent.com/1907276/157064795-f2a9688a-719f-4402-a7a5-26aac5c8c9c6.jpg)

![error-media-editor](https://user-images.githubusercontent.com/1907276/157064625-5ceceea8-c0a8-494a-879f-a5ba0c76d1ec.jpg)

Property 'mime_type' of argument 'item' of event onFetchMediaItems has a wrong item. Valid: string

**Comment by sonvnn** [comment]
Hi @Chacapamac ,

I update you as an Administrator now. Sorry for delay! I weak in 14 days by Covid-19. 

Kind Regards,
Sonny

**Reply by Chacapamac** [reply]
**Get well Sonny!**
 Me to I’m less active because I need to open a big website in few days, after that, I will be back full steam!

Thanks for the administrative right.
**Be sure that I will NEVER do anything before getting your approbation here first.**
